src/routes/settings/+page.svelte

```diff
@@ -107,8 +108,10 @@
 		{#if $activeProfile?.privkey != undefined}
 			<label>PK: <input bind:value={pk} disabled /></label><br />
 			<label>Name: <input bind:value={name} /></label><br />
+			<div class="avatar">
+				<AvatarImage profile={$activeProfile} />
```

Comment:
The settings page did show the source image rather than how it would look like. It makes sense to change that but now it's not quite finished:

![Screenshot from 2022-10-21 13-26-24](https://user-images.githubusercontent.com/96378/197245392-b39c9719-aa3b-41e1-a685-75d9cfa589b9.png)

This should be a circle, not an ellipsis.